msg/vehicle_odometry.msg

```diff
@@ -0,0 +1,34 @@
+# Vehicle odometry data. Fits ROS REP 147 for aerial vehicles
+uint64 timestamp		# time since system start (microseconds)
+
+# Position in NED earth-fixed frame (meters). NaN if invalid/unknown
```

Comment:
@TSC21 is this really always aligned with NED? (aligned with north, east ?)

Reply:
In the case of PX4, I think this is correct, yes.

Reply:
No this is not correct in most cases, since most external VO systems do not have a global yaw reference, and neither is the frame necessarily earth-fixed.

This is the reason the EKF2 options exist to either : 
1. Use vision yaw as a fake yaw aligned to true north (prevents use of GPS or other sources actually operating in global frame)
Or, 
2. Estimate the transform between the vision frame and an earth-fixed NED frame online.

The correct comment here should be "position in a right-handed local frame (FRD?) ".